x-pack/plugins/transform/public/app/sections/transform_management/components/edit_transform_flyout/edit_transform_flyout_form.tsx

```diff
@@ -33,36 +33,130 @@ export const EditTransformFlyoutForm: FC<EditTransformFlyoutFormProps> = ({
+          }
+        )}
         value={formFields.frequency.value}
       />
+
+      <EuiSpacer size="l" />
+
+      <EuiAccordion
+        data-test-subj="transformEditAccordionDestination"
+        id="transformEditAccordionDestination"
+        buttonContent={i18n.translate(
+          'xpack.transform.transformList.editFlyoutFormDestinationButtonContent',
+          {
+            defaultMessage: 'Destination configuration',
+          }
+        )}
+        paddingSize="s"
+      >
+        <EditTransformFlyoutFormTextInput
+          dataTestSubj="transformEditFlyoutDestinationIndexInput"
+          errorMessages={formFields.destinationIndex.errorMessages}
+          label={i18n.translate(
+            'xpack.transform.transformList.editFlyoutFormDestinationIndexLabel',
+            {
+              defaultMessage: 'Destination index',
+            }
+          )}
+          onChange={(value) => dispatch({ field: 'destinationIndex', value })}
+          value={formFields.destinationIndex.value}
+        />
+
+        <EditTransformFlyoutFormTextInput
+          dataTestSubj="transformEditFlyoutDestinationPipelineInput"
+          errorMessages={formFields.destinationPipeline.errorMessages}
+          label={i18n.translate(
+            'xpack.transform.transformList.editFlyoutFormDestinationPipelineLabel',
+            {
+              defaultMessage: 'Pipeline',
+            }
+          )}
+          onChange={(value) => dispatch({ field: 'destinationPipeline', value })}
+          value={formFields.destinationPipeline.value}
+        />
+      </EuiAccordion>
+
+      <EuiSpacer size="l" />
+
+      <EuiAccordion
+        data-test-subj="transformEditAccordionAdvancedSettings"
+        id="transformEditAccordionAdvancedSettings"
+        buttonContent={i18n.translate(
+          'xpack.transform.transformList.editFlyoutFormAdvancedSettingsButtonContent',
+          {
+            defaultMessage: 'Advanced settings',
+          }
+        )}
+        paddingSize="s"
+      >
+        <div data-test-subj="transformEditAccordionAdvancedSettingsContent">
+          <EditTransformFlyoutFormTextInput
+            dataTestSubj="transformEditFlyoutDocsPerSecondInput"
+            errorMessages={formFields.docsPerSecond.errorMessages}
+            helpText={i18n.translate(
+              'xpack.transform.transformList.editFlyoutFormDocsPerSecondHelptext',
+              {
+                defaultMessage:
+                  'To enable throttling, set a limit of documents per second of input documents.',
+              }
+            )}
+            label={i18n.translate(
+              'xpack.transform.transformList.editFlyoutFormDocsPerSecondLabel',
+              {
+                defaultMessage: 'Documents per second',
+              }
+            )}
+            onChange={(value) => dispatch({ field: 'docsPerSecond', value })}
+            value={formFields.docsPerSecond.value}
+          />
+
+          <EditTransformFlyoutFormTextInput
```

Comment:
Is it possible to only allow integer input in these fields?

![image](https://user-images.githubusercontent.com/7405507/93220297-3703af00-f764-11ea-96b7-0c2ac105b85a.png)

Reply:
Improved validation to check if an integer and updated unit tests in 0892877975e.